### DISCUSSION TITLE
Loading Hugginface Models - but not all from master

### DISCUSSION THREAD

**Discussion by Aevermann** [oobabooga/text-generation-webui#3335] [Q&A]
How can I limit the download of a Hugginface model with a master tree that contains a lot of models to only downloading a specific version? In my case the master tree contains ~ 200GB of models but I only want to load one of them.

**Comment by missionfloyd** [comment]
GGML models like [this](https://huggingface.co/TheBloke/Nous-Hermes-Llama-2-7B-GGML/tree/main)? Manually download the one you want and put in the models folder.
